### PR TITLE
[LCS-280] New page /privacy-policy replace T&Cs

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
Also add privacy-policy to the footer of the page

### CODE REVIEW

File: apps/right-to-rent-check/acceptance/features/privacy-policy.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const steps = require('../../');
+
+Feature('Given I am on /privacy-policy');
+
+Before((
+  I,
+  privacyPolicyPage
+) => {
+  I.visitPage(privacyPolicyPage, steps);
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/acceptance/features/privacy-policy.js]: Is it ok to use visitPage in this step rather than completeToStep?

Comment (lennym) [reply apps/right-to-rent-check/acceptance/features/privacy-policy.js]: I personally don't see any point in functional testing static pages.

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/acceptance/features/privacy-policy.js]: I agree, I think it just adds more time to run tests and more time to maintain.  I'll remove it.

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/acceptance/features/privacy-policy.js]: I've now updated

File: apps/right-to-rent-check/acceptance/pages/privacy-policy.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  url: 'privacy-policy'
+};
```
Comment (lennym) [inline apps/right-to-rent-check/acceptance/pages/privacy-policy.js]: This file needs deleting.

**Pull request merged**